src/pranitaPJun20/MeetingRoomOverloading.java

```diff
@@ -0,0 +1,67 @@
+package pranitaPJun20;
+
+public class MeetingRoomOverloading {
+	int bookingTime;
+	String selectedMeetingRoom;
+	String[] rooms = { "Nalanda", "Takshashila", "AgraFort", "PratapGadh" };
+
+	void bookMeetingRoom() {
+		selectedMeetingRoom = "Nalanda";
+		bookingTime = 1;
+		displayInfo();
+		for (int index = 0; index < rooms.length; index++) {
+			if (rooms[index].equals("Nalanda")) {
+				rooms[index] = " ";
+			}
+		}
+	}
+
+	void bookMeetingRoom(String meetingRoomName) {
+		if (!meetingRoomName.equals("Nalanda")) {
+			selectedMeetingRoom = meetingRoomName;
+			bookingTime = 1;
+			for (int index = 0; index < rooms.length; index++) {
+				if (rooms[index].equals(meetingRoomName)) {
+					rooms[index] = " ";
+				}
+			}
+			displayInfo();
+		} else {
+			System.out.println("\nThe meeting room Nalanda is already booked.Please book any other meeting room");
+		}
+
+	}
+
+	void bookMeetingRoom(String meetingRoomName, int time) {
+		selectedMeetingRoom = meetingRoomName;
+		bookingTime = time;
+		for (int index = 0; index < rooms.length; index++) {
+			if (rooms[index].equals(meetingRoomName)) {
+				rooms[index] = " ";
+			}
+		}
+		displayInfo();
+	}
+
+	void displayInfo() {
+		System.out.println("\nMeeting room " + selectedMeetingRoom + " is booked for " + bookingTime + " hour");
+	}
+
+	void displayAvilableMeetingRoomForBooking() {
+		for (int index = 0; index < rooms.length; index++) {
+			if (!rooms[index].equals(" "))
+				System.out.println("\nAvailable meeting rooms : " + rooms[index]);
+		}
+
+	}
+
+	public static void main(String[] args) {
+		MeetingRoomOverloading meetingRoomOverloading = new MeetingRoomOverloading();
+		meetingRoomOverloading.bookMeetingRoom();
+		meetingRoomOverloading.bookMeetingRoom("Nalanda");
+		meetingRoomOverloading.bookMeetingRoom("Takshashila");
+		meetingRoomOverloading.bookMeetingRoom("AgraFort", 1);
```

Comment:
here you call displayInfo method in every method, i want that method call by customer to check which is booked for how many hours, for that check below hint:

what you should do is define String array named as meetingRoomName (already in your code) and second array which contains hours.
when someone booked room that time you should update the same index value.

in displaInfor() you will use for loop and check, if hours is 0 then on same index room is also available.

String[] meetingRoomName = {"Nalanda", "Takshashila", "AgraFort","PratapGadh"};
int hours[] = {0,0,0,0};

this hint may help you to think on this. for now we know array which can store multiple value, so think on this
before book any room, in your code you should add method named as isRoomAvailable() [this method checks on hours array if any slot is available than return true to book room]

Reply:
Thank you for the detailed explanation.I uploaded new files with all code changes.Please review that.